README.md

```diff
@@ -10,12 +10,29 @@ The goal of this project is to make feature complete Gothic client-app compatibl
 [![Build status](https://ci.appveyor.com/api/projects/status/github/Try/opengothic?svg=true)](https://ci.appveyor.com/project/Try/opengothic)  
 Core gameplay is done, you can complete first chapter, as well as all addon content for any guild. 
 
-##### How to play
+##### Install on Windows
 1. Install original gothic game from CD/Steam/GOG/etc  
 *you have to install original game, since OpenGothic does not have any game assets or game scripts as built-in*
 2. [Download latest stable build](https://github.com/Try/opengothic/releases/latest)
 3. run '/OpenGothic/bin/Gothic2Notr.exe -g "C:\Program Files (x86)\Path\To\Gothic II"'
 
+##### Build it for Linux
+1. Install dependencies:
+
+* for Ubuntu:
+`sudo apt install cmake g++ doxygen glslang-tools libgl-dev`
+
+* for openSUSE:
+`sudo zypper in git cmake gcc-c++ glslang-devel doxygen Mesa-devel`
```

Comment:
Doxygen for opensuse? You don't really need it to compile code, right?

Library `libasound2-dev` also reqired to build sound.
`libvulkan-dev` is not in list, what is odd for Vulkan project.
`libgl-dev`, 'Mesa-devel' - I thought, redundant dependency from those libraries was fixed a sometime ago. Do we still need those?

Reply:
I'll test this section on a clean VM later, but it's true that with these build dependencies the cmake command does not output "not found".
On Tumbleweed I believe Mesa-devel pulls all the required openGL libraries, but I'll test it again.

Reply:
I mean: since project not using opengl, there not suppose to be any gl-packages, right?
The engine requires glslangValidator, but it suppose to be included in vulkan-sdk,